how-to-contribute.md

```diff
@@ -1,177 +1,198 @@
 
-The Shiro Project is an [Open Source](https://opensource.org/) volunteer project released under a [very liberal license](license.html "License"). This means there are many ways to contribute to the project - either with direct participation (coding, documenting, answering questions, proposing ideas, reporting bugs, suggesting bug-fixes, etc..) or by resource donations (staff time, conference presentations, publicity, software) and even general hardware/money [donations](http://www.apache.org/foundation/thanks.html) via the [Apache Software Foundation](http://www.apache.org).
+There are many ways how to contribute to this project, you can participate directly by:
+* coding
+* documenting
+* answering questions
+* proposing ideas
+* reporting bugs
+* suggesting bug-fixes
+* etc.
 
-To begin with, we suggest you to subscribe to the [Shiro mailing lists](mailing-lists.html "Mailing Lists") (follow the link for information on how to subscribe and to access the mail list archives). Listen-in for a while, to hear how others make contributions.
+Or you can participate with resource donations:
+* staff time
+* conference presentations
+* publicity
+* software
+* general hardware/money [donations](http://www.apache.org/foundation/thanks.html) via the [Apache Software Foundation](http://www.apache.org)
 
-You can get your local working copy of the [latest and greatest code](download.html "Download") by following the directions in our [Download](download.html "Download") page. Review the To Do list in the [issue tracker](https://issues.apache.org/jira/browse/SHIRO) and then choose a task that interests you. Perhaps you have noticed something that needs patching, or have a new feature to contribute. Make the changes, do the testing, generate a patch, and discuss on the [dev mailing list](mailing-lists.html "Mailing Lists"). (Do not worry - the process is easy and explained below.)
+You can get your local working copy of the [latest and greatest code](download.html "Download") by following the directions in [Download](download.html "Download") page. Review the To Do list in the [issue tracker](https://issues.apache.org/Jira/browse/SHIRO) and then choose a task that interests you.
 
-Document writers are usually the most wanted people so if you like to help but you're not familiar with the innermost technical details, don't worry: you can still be tremendously helpful!
+Perhaps you have noticed something that needs patching or have a new feature to contribute. Make the changes, do the testing, generate a patch, and discuss on the [dev mailing list](mailing-lists.html "Mailing Lists"). The process is easy and explained below. Subscribe to the mailing list!
```

Comment:
Add a real link here (https://shiro.apache.org/mailing-lists.html)

Reply:
Fixed this.

But... The real link is missing approx from twenty *.md files from shiro-site project. Only mailing.lists.html is used on those files instead of the real link. FYI @bmarwell .